docs/UserGuide.md

```diff
@@ -128,9 +142,27 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 * `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
+* `find alex david` returns `Alex Yeoh` and `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Filtering persons by tag: `filter`
+
+Finds persons that are tagged with any of the given tags.
+
+Format: `filter TAG [MORE_TAGS]`
+
+* The search is case-insensitive. e.g `friends` will match with `Friends`.
+* The order of the tags does not matter. e.g. `friends family` will match `family friends`.
+* Only the tag is searched.
+* Only full tags will be matched e.g. `friend` will not match `friends`
```

Comment:
Maybe add period on line 157 for consistency.

Reply:
Added period on all sentences.